src/jsifier.js

```diff
@@ -426,6 +426,7 @@ function ${name}(${args}) {
       }
       if (isStub) {
         contentText += `\n${mangled}.stub = true;`;
+        contentText += `\nwasmImportStubs['${symbol}'] = undefined;`;
```

Comment:
Why do we need to set it to undefined?

Reply:
So that `library_dylink.js` knows to full this in with the real symbol later.

I will add some comments about this.